inclusion_connect/settings/base.py

```diff
@@ -364,3 +364,12 @@
 
 CSP_INCLUDE_NONCE_IN = ["script-src"]
 CSP_REPORT_URI = os.getenv("CSP_REPORT_URI", None)
+
+# CORS
+# ----
+
+CORS_ALLOW_ALL_ORIGINS = os.getenv("CORS_ALLOW_ALL_ORIGINS") == "True"
+
+cors_allowed_origins = os.getenv("CORS_ALLOWED_ORIGINS")
+if cors_allowed_origins:
+    CORS_ALLOWED_ORIGINS = cors_allowed_origins.split(",")
```

Comment:
Ce setting sera désactivé si `CORS_ALLOW_ALL_ORIGINS == True`. Peut-être ?

```suggestion
if cors_allowed_origins and not CORS_ALLOW_ALL_ORIGINS:
    CORS_ALLOWED_ORIGINS = cors_allowed_origins.split(",")
```

Reply:
`CORS_ALLOW_ALL_ORIGINS` : `If True, all origins will be allowed. Other settings restricting allowed origins will be ignored. Defaults to False.`
cf https://pypi.org/project/django-cors-headers/
Donc pas besoin de mettre de conditions supplémentaires, si ce n'est pour rendre clair que ce n'est pas utilisé avec `CORS_ALLOW_ALL_ORIGINS`.

Reply:
Ah, je crois qu'on est d'accord et que c'est une suggestion pour que ce soit clair dans le base.py, ça me va :+1: